index.html

```diff
@@ -2370,13 +2370,41 @@ <h3>Unlinkability</h3>
       <section>
         <h3>Selective Disclosure</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are selective
-disclosure digital signature schemes, such as BBS+, that are capable of
-not disclosing correlatable identifiers and ensuring that a different
-but valid digital signature is re-created upon every presentation.
+        <p>
+Selective disclosure is a technique that enables the recipient of a
+previously-signed message (that is, a message signed by its creator) to reveal
+parts of the message without revealing the entire message and without disturbing
+the verifiability of the origin of the parts being revealed. For example,
+selectively disclosing a digital driver's license for the purpose of renting a
+car might involve the entity presenting the license to only reveal the issuing
+authority, license number, birthday, and authorized motor vehicle class. Note
+that in this case, the license number is correlatable information that is being
+shared but some amount of privacy is still being preserved because the driver's
+full name and address are not being shared.
+        </p>
+
+        <p>
+The creator of an initial message that they expect to be selectively disclosed
+needs to use a capable cryptosuite and enable selective disclosure on the
+message in order for a recipient of that message to use the technique on future
+disclosures. The creator might also need to make it mandatory to disclose
+certain parts of a message. The recipient of the message that desires to
+selectively disclose the original message needs to then utilize software that is
+capable of performing selective disclosure. Not all software or cryptosuites are
+capable of providing selective disclosure. An example of a cryptosuite that does
+support selective disclosure is `bbs-2022`.
```

Comment:
I had a hard time parsing this... here's my attempted rewrite:

```suggestion
Not all software or cryptosuites are capable of providing selective disclosure.
If the author of a message wishes it to be selectively disclosable by
its recipient, then they need to both use a capable cryptosuite and enable
selective disclosure on the specific message. They might also make it mandatory
to disclose certain parts of the message. A recipient that wants to selectively
disclose the message needs to utilize software that is able to perform the
technique. An example of a cryptosuite that supports selective disclosure is
`bbs-2022`.
```